docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md

```diff
@@ -97,6 +104,12 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 In this step, you will configure the yaml required for HAProxy Collection.
 
+Below are the inputs required:
+
+- **Endpoint**. The URL of the httpd status endpoint (default: `http://localhost:8404/stats`).
+- **HAProxy logs Path**. Enter the path to the log file for your HAProxy instance.
+- **Fields**. `webengine.cluster.name`.
```

Comment:
This line is not making any sense. Can we remove it or provide more information

Reply:
removed it.